#Scripts and their hierarchy:
#b. pipeline.sh: runs Fastp to identify adapter sequence and quality trim.
#c. alignment.sh: runs bowtie1 to perform alignment to miRbase reference genome.
#d. counting.sh: used for quantification to count reads that aligned to miRNA.
#e. countextract.sh: used to extract all miRNA from the human genome.
#f. expressionanalysis.R: performs normalization and gene differential expression.
![image](https://github.com/nsybee/BioinformaticsAnalysis/assets/115989775/7e4b566c-cf2c-46a9-bdcc-be0cf00dc98d)


Note all the zip folders has to be unzipped

